src/frontend-admin/src/components/App/App.tsx

```diff
@@ -20,6 +22,62 @@ export const App: React.FC = () => {
           Learn React
         </a>
       </header>
-    </div>
+  </div>*/
```

Comment:
Totally ok to delete code that's no longer needed - that's why we have git in the first place! :)